frontend/views/top_containers/bulk_operations/_bulk_action_templates.html.erb

```diff
@@ -4,4 +4,90 @@
     <li><strong>${container[1]}</strong>&nbsp;&nbsp;&nbsp;${container[0]}</li>
   {/for}
   </ul>
---></div>
\ No newline at end of file
+--></div>
+
+<div id="bulk_action_update_ils_fields"><!--
```

Comment:
I was wondering if we should show a preview of the user's selection on these templates to serve as a kind of confirmation.  An alternative is an actual confirmation page?  

As we have the 'jsonmodel type' and display string we're able to render the token view of the records.  This could be displayed either above or to the left of the form in a scrollable panel.